Exiled.API/Features/DamageHandler.cs

```diff
@@ -140,51 +139,31 @@ public DamageType Type
                             return DamageType.Explosion;
                         case Scp018DamageHandler _:
                             return DamageType.Scp018;
                         case RecontainmentDamageHandler _:
                             return DamageType.Recontainment;
+                        case DisruptorDamageHandler _:
+                            return DamageType.MolecularDisruptor;
                         case UniversalDamageHandler universal:
-                        {
-                            DeathTranslation translation = DeathTranslations.TranslationsById[universal.TranslationId];
-
-                            if (translationConversion.ContainsKey(translation))
-                                return translationConversion[translation];
-                            if (translation.Id == DeathTranslations.Asphyxiated.Id)
-                                return DamageType.Asphyxiation;
-                            if (translation.Id == DeathTranslations.Bleeding.Id)
-                                return DamageType.Bleeding;
-                            if (translation.Id == DeathTranslations.Decontamination.Id)
-                                return DamageType.Decontamination;
-                            if (translation.Id == DeathTranslations.Poisoned.Id)
-                                return DamageType.Poison;
-                            if (translation.Id == DeathTranslations.Falldown.Id)
-                                return DamageType.Falldown;
-                            if (translation.Id == DeathTranslations.Tesla.Id)
-                                return DamageType.Tesla;
-                            if (translation.Id == DeathTranslations.Scp207.Id)
-                                return DamageType.Scp207;
-                            if (translation.Id == DeathTranslations.Crushed.Id)
-                                return DamageType.Crushed;
-                            if (translation.Id == DeathTranslations.UsedAs106Bait.Id)
-                                return DamageType.FemurBreaker;
-                            if (translation.Id == DeathTranslations.FriendlyFireDetector.Id)
-                                return DamageType.FriendlyFireDetector;
-                            if (translation.Id == DeathTranslations.SeveredHands.Id)
-                                return DamageType.SeveredHands;
-
-                            Log.Warn($"{nameof(DamageHandler)}.{nameof(Type)}: No matching {nameof(DamageType)} for {nameof(UniversalDamageHandler)} with ID {translation.Id}, type will be reported as {DamageType.Unknown}. Report this to EXILED Devs.");
-                            break;
-                        }
+                            {
+                                DeathTranslation translation = DeathTranslations.TranslationsById[universal.TranslationId];
+
+                                if (translationConversion.ContainsKey(translation))
```

Comment:
We should still do ID checks. I'm honestly not very trusting of Northwood to not make a new instance of translations in some places, since they did that with the old DamageTypes.

Reply:
Should I make 20 ifs for every single damage type then?

Reply:
Because again, I don't see a reason since the dictionary exists and if new damage type is added it will fail anyway no matter if there are ifs or not.

Reply:
Whether or not we do the if-return tree comes down to if we trust NW or not, to continue using their dictionary properly.